docs_redirects/app.py

```diff
@@ -0,0 +1,128 @@
+    "/getting-started/quickstart": base + "getting-started",
+    "/getting-started/reference": base + "reference",
+    "/getting-started/troubleshooting": base + "troubleshooting",
+
+    "/import-export/clipboard": base + "clipboard",
+    "/import-export/export-data-as-csv": base + "export-data-csv",
+
+    "/layout/borders": base + "styling-borders",
+    "/layout/cell-styling": base + "styling-cells",
+    "/layout/color-font": base + "styling-color-font",
+    "/layout/grid-size": base + "grid-size",
+    "/layout/headers": base + "styling-headers",
+    "/layout/row-styling": base + "styling-rows",
+    "/layout/selections": base + "styling-selections",
+    "/layout/themes": base + "styling-themes",
+
+    "/other-examples/crossfilter": base + "",                     # This page doesn't exist yet in dash-docs
+    "/other-examples/popup-from-cell-click": base + "popups",
+    "/other-examples/virtual-row-data": base + "",                 # This page doesn't exist yet in dash-docs
+
+    "/persistence/persistence": base + "persistence",
+
+    "/rendering/change-cell-renderers": base + "change-cell-renderers",
+    "/rendering/value-formatters-custom-functions": base + "custom-functions-value-formatters",
+    "/rendering/value-formatters-intro": base + "value-formatters",
+    "/rendering/value-formatters-with-d3-format": base + "d3-value-formatters",
+    "/rendering/value-getters": base + "value-getters",
+
+    "/rows/row-dragging": base + "row-dragging",
+    "/rows/row-height": base + "row-height",
+    "/rows/row-ids": base + "row-ids",
+    "/rows/row-pinning": base + "row-pinning",
+    "/rows/row-sorting": base + "row-sorting",
+    "/rows/row-spanning": base + "row-spanning",
+
+    "/scrolling/aligned-grids": base + "aligned-grids",
+    "/scrolling/infinite-scroll": base + "infinite-scroll",
+    "/scrolling/pagination": base + "pagination",
+    "/scrolling/scroll-to": base + "scroll-to",
+    "/scrolling/scrolling-performance" : base + "pagination",      # This page doesn't exist yet in dash-docs
+
+    "/selection/cell-selection": base + "cell-selection",
+    "/selection/range-selection": base + "enterprise-ag-grid",
+    "/selection/select-everything-or-filtered" : base + "",        # This page doesn't exist yet in dash-docs
+    "/selection/selection-checkbox": base + "",
+    "/selection/selection-multiple-click": base + "row-selection-multiple-on-click",
+    "/selection/selection-multiple": base + "row-selection-multiple",
+    "/selection/selection-single": base + "row-selection-single",
+    "/selection/text-selection": base + "",
+
+    "/serverside-data/infinite-row-model": base + "infinite-row-model",
+    "/serverside-data/row-models": base + "row-models",
+
+}
+
+app = Dash(__name__)
+
+app.layout =  html.Div("no layout")
+
+@app.server.before_request
+def redirect():
+    print(flask.request.path)
+    return flask.redirect(redirect_map.get(flask.request.path))
```

Comment:
```suggestion
    return flask.redirect(redirect_map.get(flask.request.path, base))
```
In case there's a link out there that doesn't match one of these pages exactly (try visiting https://dashaggrid.pythonanywhere.com/doesnotexist right now - you get an infinite redirect loop to https://dashaggrid.pythonanywhere.com/None)